content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,203 @@
+=================
+PDF quote builder
+=================
+
+The *PDF Quote Builder* in Odoo *Sales* provides the opportunity to export a fully-custom PDF file
+for quotes, showcasing various information and design elements, instead of just the price and total.
+
+The PDF Quote Builder can be used to add header pages, product descriptions, and footer pages to a
+quote.
+
+Having a customized PDF in quotes provides a heightened conclusion to the shopping experience for
+customers, and adds an elegant level of professionalism to a company.
+
+Configuration
+=============
+
+In order to add custom PDF files for quotes, the :guilabel:`PDF Quote builder` feature *must* be
+configured.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`. Then, on the
+:guilabel:`Settings` page, scroll to the :guilabel:`Quotations & Orders` section, and locate the
+:guilabel:`PDF Quote builder` feature.
+
+.. image:: pdf_quote_builder/pdf-quote-builder-feature.png
+   :align: center
+   :alt: The PDF Quote Builder feature located on the Settings page of the Sales application.
+
+Here, custom :guilabel:`Header pages` and :guilabel:`Footer pages` can be uploaded. To upload
+either, click the :guilabel:`Upload your file` button, or the :guilabel:`🖉 (pencil)` icon to the
+right of the desired field, and proceed to locate, select, and upload the desired PDF file from the
+computer's hard drive.
+
+Clicking the :guilabel:`🗑️ (trash)` icon deletes the current PDF file, and replaces the blank field
+with an :guilabel:`Upload your file` button. When :guilabel:`Upload your file` is clicked, proceed
+to upload the desired PDF files, using the same process detailed above.
+
+Once the desired PDF file(s) are uploaded in the appropriate fields in the :guilabel:`PDF Quote
+builder` section of the *Sales* :guilabel:`Settings` page, be sure to click :guilabel:`Save`.
+
+The files uploaded here will be the default PDF used for all quotes.
+
+.. note::
+    Values set here are company-specific.
+
+Dynamic text in PDFs
+====================
+
+While creating custom PDFs for quotes, using any standard PDF editor software, certain dynamic texts
+should be utilized throughout the design of the PDF to ensure Odoo can automatically fill in the
+appropriate content accordingly.
+
+Dynamic text values are, essentially, coded placeholders that Odoo fills in automatically, according
+to the specific information related to the quote.
+
+Dyanmic text values
+-------------------
+
+Below are the dynamic text values and what they will represent in the custom PDFs:
+
+- :guilabel:`name`: Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
+
```

Comment:
I completely understand your logic here. However, I think this may be a bit "disconnected" from the overall concept of this doc, aimed specifically at PDFs (not emails, per se). That said, again, I fully get where you're coming from with this. So, I think I'll add a quick visual example of a PDF using dynamic text placeholders to showcase a more doc-appropriate sample of how *that* would look. Hope that will suffice. Thanks for bringing this to my attention. I really appreciate your thoughtfulness (and thoroughness) with your suggestions. 👍

Reply:
Aww (: Thanks for taking my feedback seriously.The screenshot is perfect, and I now have a clearer understanding of how dynamic text is used in a PDF. Thank you for addressing my questions!